engines/ags/plugins/ags_waves/sound.cpp

```diff
@@ -248,9 +242,8 @@ void AGSWaves::MusicPlay(int MusicToPlay, int repeat, int fadeinMS, int fadeoutM
 	_mixer->stopHandle(MFXStream._soundHandle);
 
 	// Load OGG file for music
-	Common::FSNode fsNode = ::AGS::g_vm->getGameFolder().getChild(
-		"Music").getChild(Common::String::format("music%d.mfx", MusicToPlay));
-	Audio::AudioStream *musicStream = loadOGG(fsNode);
+	Common::ArchiveMemberPtr member = getFile(Common::String::format("music/music%d.sfx", MusicToPlay).c_str());
```

Comment:
```suggestion
	Common::ArchiveMemberPtr member = getFile(Common::String::format("music/music%d.mfx", MusicToPlay).c_str());
```

Reply:
Oops! This was a bad copy-paste. Thanks!